main.tf

```diff
@@ -1,4 +1,4 @@
-resource "commercetools_api_client" "unprotected" {
+resource "commercetools_api_client" "api_client" {
```

Comment:
Why not simply rotate the credentials?

Reply:
This has added unnecessary complexity to already complex process for migration from 0.12 to 0.13.
I would rather avoid additional work. Nobody checked effect of this on this scale + under high load.

Reply:
Not sure how relevant the internal affairs are for the general public :)

Reply:
@danvaida updated

Reply:
Indeed, it will force the rotation of the existing credentials.
I think this is an acceptable outcome, not to mention that it is a security best-practice.

Reply:
After careful consideration of the internal implementation and all the known limitations of it, I am approving it.